Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM node:17-alpine
 
 ARG NEXT_PUBLIC_GRAPHQL_ENDPOINT
+ARG NEXT_PUBLIC_RIBBON_VERSION
```

Comment:
I suggest we make `SENTRY_RELEASE` more generic, call it `RELEASE`, and use it for both Sentry and the ribbon. What do you think @qgerome ?